code/WorkInProgress/griffening/griffening_cards.dm

```diff
@@ -56,8 +56,40 @@
 #define GRIFFENING_ATTRIBUTE_HEAD 1024
 #define GRIFFENING_ATTRIBUTE_DEFAULT GRIFFENING_ATTRIBUTE_HANDS | GRIFFENING_ATTRIBUTE_ORGANIC | GRIFFENING_ATTRIBUTE_HUMAN
 
+/datum/playing_card
+	var/card_name = "playing card"
+	var/card_desc = "A card, for playing some kinda game with."
+	var/card_face = "blank"
+	var/card_back = "suit"
+	var/card_foil = 0
+	var/card_data = null
+	var/card_reversible = 0 // can the card be drawn reversed? ie for tarot
+	var/card_reversed = 0 // IS it reversed?
+	var/card_tappable = 1 // tap 2 islands for mana
+	var/card_tapped = 0 // summon Fog Bank, laugh
+	var/card_spooky = 0
+	var/solitaire_offset = 3
+
+	New(cardname, carddesc, cardback, cardface, cardfoil, carddata, cardreversible, cardreversed, cardtappable, cardtapped, cardspooky, cardsolitaire)
+		..()
+		if (cardname) src.card_name = cardname
+		if (carddesc) src.card_desc = carddesc
+		if (cardback) src.card_back = cardback
+		if (cardface) src.card_face = cardface
+		if (cardfoil) src.card_foil = cardfoil
+		if (carddata) src.card_data = carddata
+		if (cardreversible) src.card_reversible = cardreversible
+		if (cardreversed) src.card_reversed = cardreversed
+		if (cardtappable) src.card_tappable = cardtappable
+		if (cardtapped) src.card_tapped = cardtapped
+		if (cardspooky) src.card_spooky = cardspooky
+		if (cardsolitaire) src.solitaire_offset = cardsolitaire
+
+	proc/examine_data()
+		return card_data
+
 /datum/playing_card/griffening
-	proc/enter_play(var/datum/griffening_controller/field_data, var/active_player)
+	/*proc/enter_play(var/datum/griffening_controller/field_data, var/active_player)
```

Comment:
don't just comment out code....
keep it, delete it, or move it to some un-included file

Reply:
this code wasn't mine, so i didn't want to just...erase it forever. i'm not sure who the original creator is or if they plan on coming back to it, so i just commented it out because i was unsure of what to do

Reply:
it's marq, he's not likely to come back to coding

Reply:
okies, thank you